hyo2/ssm2/lib/client/client.py

```diff
@@ -51,14 +51,17 @@ def send_kng_format(self, prj: 'SoundSpeedLibrary', server_mode: bool = False) -
         if (self.protocol == "QINSY") or (self.protocol == "PDS2000"):
             kng_fmt = Dicts.kng_formats['S12']
             logger.info("forcing S12 format")
+        if (self.protocol == "EA440"):
+            kng_fmt = Dicts.kng_formats['S01']
+            logger.info("forcing S01 format")
 
         apply_thin = True
         apply_12k = True
         tolerances = [0.01, 0.03, 0.06, 0.1, 0.5]
         if self.protocol == "QINSY":
             apply_12k = False
             tolerances = [0.001, 0.005, 0.01, 0.05, 0.1, 0.5]
-        elif self.protocol == "PDS2000":
+        elif (self.protocol == "PDS2000") or (self.protocol == "EA440"):
```

Comment:
Why do we need `apply_12k = False` for the `EA440` protocol?

Reply:
The problem with the 12k extension is that you'll see the following when surveying in shallow water:
![12k_True](https://github.com/user-attachments/assets/17126a43-c38b-493a-9cfb-4a4b6e23c8ad)

Since the EA440 software does not provide a reception confirmation, the only way that the surveyor can confirm reception of the transmitted cast is by visual inspection. In the EA440 display, I did not find a way to adjust the limits of the Y-axis. So basically, the surveyor always see the same graph regardless of which cast is sent.

Does this justify breaking the 12k extension in your view?

Reply:
Yes, it makes sense. Maybe a comment in the code to avoid that we forget about it?